src/utils/getHumanCapacity.ts

```diff
@@ -1,9 +1,11 @@
-import { find, round } from "lodash";
-import { GiB } from "../constants";
+import { find } from "lodash";
+import prettyBytes from "pretty-bytes";
 import type { Maybe } from "../types";
 import type { DiskPartition } from "../services/lansweeper/types";
 
-const bytesToGB = (bytes: number) => bytes / GiB;
+const options = { minimumFractionDigits: 2, maximumFractionDigits: 2 };
```

Comment:
Why set `minimumFractionDigits` to `2`. `0` is probably better because adding trailing `0`s adds no value.

Reply:
removed `minimumFractionDigits`